pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -704,7 +700,6 @@ func (o *MustGatherOptions) newPod(node, image string) *corev1.Pod {
 			TerminationGracePeriodSeconds: &zero,
 			Tolerations: []corev1.Toleration{
 				{
-					Key:      tolerationKey,
```

Comment:
This is supposed to be blanket toleration, now we changed it to just master :(. 

@@apoorvajagtap
If you want something like you should list all the taints that you would like to see tolerations against

Reply:
Hi @ravisantoshgudimetla my perspective was more inclined to the fact that by default we do not have any custom taints added to master nodes, and considering that when `--node-name` is not specified the pod would schedule on one of the master nodes, I opted for this single toleration.
But, now when I think of it, I agree if there exists a cluster that's not in an ideal situation, this might be a problem. Let me know in case I am missing anything else too.